Neuter git(1) usage, replace timestamp for with static "OpenBSD" string

Index: singleheader/amalgamate.py
--- singleheader/amalgamate.py.orig
+++ singleheader/amalgamate.py
@@ -98,9 +98,7 @@ def dofile(fid: str, prepath: str, filename: str) -> N
 # between gnu date and busybox date.
 try:
     timestamp = (
-        subprocess.run(['git', 'show', '-s', '--format=%ci', 'HEAD'], stdout=subprocess.PIPE)
-        .stdout.decode('utf-8')
-        .strip()
+        'OpenBSD'
     )
 except Exception:
     print('git not found, timestamp based on current time')
